src/components/flyout/_variables.scss

```diff
@@ -0,0 +1 @@
+$overlay-header-height: 64px;
```

Comment:
I think we'll want to use SUIT naming `$OverlayHeader-height` here? Although I don't think we've previously done that in this repo for scss variables so could get a second opinion on this.

Reply:
hm, doesn't seem to hurt, but since these variables need to be imported, clashing doesn't seem like much of a problem

Reply:
There seems to be SUIT naming for variables 
https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md#variables

Reply:
Seems like we do use SUIT naming for variables in EUA